src/main/java/org/mockito/internal/junit/ArgMismatchFinder.java

```diff
@@ -0,0 +1,28 @@
+package org.mockito.internal.junit;
+
+import org.mockito.internal.invocation.finder.AllInvocationsFinder;
+import org.mockito.invocation.Invocation;
+import org.mockito.internal.invocation.Stubbing;
+
+/**
+ * For given mocks, finds stubbing arg mismatches
+ */
+class ArgMismatchFinder {
+
+    StubbingArgMismatches getStubbingArgMismatches(Iterable<Object> mocks) {
```

Comment:
@szczepiq 
This method should be moved to class `RuleStubbingHintsReporter` since it is only used there if I am not mistaken.